src/backend/allocator/dumb.rs

```diff
@@ -0,0 +1,79 @@
+//! Module for [DumbBuffer](https://01.org/linuxgraphics/gfx-docs/drm/gpu/drm-kms.html#dumb-buffer-objects) buffers
+
+use std::fmt;
+use std::os::unix::io::AsRawFd;
+use std::sync::Arc;
+
+use drm::buffer::Buffer as DrmBuffer;
+use drm::control::{dumbbuffer::DumbBuffer as Handle, Device as ControlDevice};
+
+use super::{Allocator, Buffer, Format};
+use crate::backend::drm::device::{DrmDevice, DrmDeviceInternal, FdWrapper};
+
+/// Wrapper around raw DumbBuffer handles.
+pub struct DumbBuffer<A: AsRawFd + 'static> {
+    fd: Arc<FdWrapper<A>>,
+    handle: Handle,
+    format: Format,
+}
+
+impl<A: AsRawFd + 'static> fmt::Debug for DumbBuffer<A> {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        f.debug_struct("DumbBuffer")
+            .field("handle", &self.handle)
+            .field("format", &self.format)
+            .finish()
+    }
```

Comment:
Why not print the fd number as well?

Reply:
Because thats the fd of the drm device. Not sure, if that is actually helpful or more confusing.